setup.py

```diff
@@ -12,7 +12,7 @@ def readme():
       long_description=readme(),
       author='Vijay Kumar B.',
       author_email='vijaykumar@bravegnu.org',
-      url='http://github.com/chennaipy/libcipher',
+      url='http://github.com/Anupama-Github/libcipher',
```

Comment:
Thank you for pointing this out. It was done to see if it related to my previous builds failure and I forgot to undo it. Will do so